### ISSUE TITLE
[BUG] Unable to install Kanister tools (kanctl/kando) from latest release - 0.21.0

### ISSUE THREAD

**Issue by pavannd1** [kanisterio/kanister#385]
**Describe the bug**
Not able to install the kanister tools - `kanctl` and `kando` using the latest release binary.
Fails on both linux and Mac OSX.

**To Reproduce**
Run the following command:
`curl https://raw.githubusercontent.com/kanisterio/kanister/master/scripts/get.sh | bash`

**Expected behavior**
`kanctl` and `kando` to be installed under `usr/local/bin`

**Screenshots**
The scripts stops running in the function `downloadFile()`.
```
Checking hash of kanister_0.21.0_darwin_amd64.tar.gz
+ pushd /var/folders/pt/sds0k5nx2wnbl_ypg10ljhp80000gn/T/kanister-installer-XXXXXX.OSrAlUqk
/var/folders/pt/sds0k5nx2wnbl_ypg10ljhp80000gn/T/kanister-installer-XXXXXX.OSrAlUqk ~/Work/Kasten/k10
+ local filtered_checksum=./kanister_0.21.0_darwin_amd64.tar.gz.sha256
+ grep kanister_0.21.0_darwin_amd64.tar.gz
pdevaraj @ ~/Work $
```

**Additional context**
From my initial debugging, the `checksum.txt` file in the release tarball seems incorrect. The sizes of the binaries themselves seems very different from previous releases.

**Comment by tdmanv** [comment]
The checksum name format changes:
![Selection_061](https://user-images.githubusercontent.com/1533907/68250198-3d1df080-ffd5-11e9-9292-69ce4d3729cd.png)
vs
![Selection_062](https://user-images.githubusercontent.com/1533907/68255185-6f354f80-ffe1-11e9-946e-2146e78d5d88.png)
 We can fix this in get.sh. I'll also check if there was a change in goreleaser

**Comment by tdmanv** [comment]
This should be fixed w/ #386
